src/styles/card.scss

```diff
@@ -0,0 +1,82 @@
+.card {
+  width: $card-width;
+  border-radius: 5px;
+  border: 1px solid $color-card-border;
+  box-sizing: border-box;
+
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    margin: 0 0 4px;
+    font-size: $font-size-card-xs;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    align-items: end;
+
+    margin-bottom: 24px;
+  }
+
+  &__code {
+    margin: 0 0 16px;
+    color: $color-secondary;
+  }
+
+  &__review {
+    margin: 0;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    font-size: $font-size-card-xs;
+    margin-bottom: 16px;
+  }
+
+  &__label {
+    color: $color-secondary;
+  }
+
+  &__cost {
+    font-size: $font-size-card-m;
+    line-height: 18px;
+    font-weight: 700;
+  }
+}
+
+.button {
+  box-sizing: border-box;
+  display: block;
+  width: 100%;
+  height: 40px;
+  margin-top: 16px;
+  border-radius: 5px;
+  background-color: $color-blue-accent;
+  color: $color-white;
+  font-size: $font-size-card-s;
+  line-height: 40px;
+  font-weight: 700;
+  text-transform: uppercase;
+  text-decoration: none;
+  text-align: center;
+  cursor: pointer;
+
+  &:hover {
+    background-color: $color-white;
+    color: $color-blue-accent;
+    border: 1px solid $color-blue-accent;
+    line-height: 38px;
```

Comment:
Better add the border for the default state of the element and just change its color on hover